core/metacling/src/TClingClassInfo.cxx

```diff
@@ -1262,7 +1262,7 @@ long TClingClassInfo::Property() const
 
    const clang::DeclContext *ctxt = GetDecl()->getDeclContext();
    clang::NamespaceDecl *std_ns =fInterp->getSema().getStdNamespace();
-   while (! ctxt->isTranslationUnit())  {
+   while (ctxt && ! ctxt->isTranslationUnit())  {
```

Comment:
(how) is this related to the patch in TCling.cxx?

Reply:
While debug the invalid state, I discover this issue (which was the actual location of the crash).   `TClingClassInfo` was not able to handle the case where it was ask to describe the global name space (empty name and has a decl with 'no' decl context).  Apply this fix, prevented the crash but did not solve the underlying issue (that we inadvertently asked for the global namespace when dealing with `std::tuple` emulation).